app/containers/ViewQuestion/index.js

```diff
@@ -288,12 +288,26 @@ export class ViewQuestion extends React.Component {
       (questionData && questionData.content.content) ||
       sendProps.translations[messages.title.description];
 
+    const articlePublishedTime =
+      questionData && questionData.post_time
+        ? new Date(questionData.post_time * 1000)
+        : ``;
+
+    const articleModifiedTime =
+      questionData && questionData.lastEditedDate
+        ? new Date(questionData.lastEditedDate * 1000)
+        : ``;
+
     return (
       <div>
-        <Helmet>
-          <title>{helmetTitle}</title>
-          <meta name="description" content={helmetDescription} />
-        </Helmet>
+        <Seo
+          title={helmetTitle}
+          description={helmetDescription}
+          language={locale}
+          keywords={helmetTitle}
```

Comment:
can we add question tags to the list of the keywords?

Reply:
sure